src/utils/arithmetic.js

```diff
@@ -0,0 +1,17 @@
+import { ERROR_TEXT, INPUT_NOT_ALLOWED_MESSAGE } from '../constants/error';
+
+export const arithmetic = (augend, addend = 0, operation) => {
```

Comment:
```suggestion
export const arithmetic = (operation, augend, addend = 0) => {
```

기본 매개변수값을 설정한 매개변수를 가장 마지막에 선언하는 것이 좋습니다. 

지금과 같은 형태에서 `addend`에 기본매개변수가 들어가게 하려면 
`arithmetic(1, undefined, "+")` 이렇게 써야하는데요, 
이렇게 쓸거면 그냥 `arithmetic(1, 0, "+")` 이렇게 쓰는게 낫겠죠 😄 

그리고 논리적인 측면에서 보더라도 전위표기법이 그렇게 이해하기 어렵지는 않은거 같아요 😄 
( `함수(변수1, 변수2)` 는 `연산자 피연산자1 피연산자2` 의 구조와 동일한만큼 그냥 함수 호출도 전위연산식과 동일한 형태니까요!)

Reply:
기본 매개변수가 없으니 못하게 되었군요..기본 매개변수를 설정할 때는 항상 유의하겠습니다!